packages/UsageAnalysis/files/Test Track/General/Files Cache.md

```diff
@@ -6,4 +6,9 @@
 6. Rename file to 'test1.txt'
 7. Rename folder to "Folder cache test1"
 8. Delete "Folder cache test1" folder. Make sure the folder is deleted.
-9. Check cache mappings, the mapping for created folder should be deleted also (Context menu, Cache...)
\ No newline at end of file
+9. Check cache mappings, the mapping for created folder should be deleted also (Context menu, Cache...)
```

Comment:
Should we validate that the cache works? Not just the settings

Reply:
We validate that the cache works by auto tests